components/chainhook-types-rs/src/rosetta.rs

```diff
@@ -8,6 +8,7 @@ use std::collections::HashSet;
 use std::fmt::Display;
 use std::hash::{Hash, Hasher};
 
+pub const DEFAULT_STACKS_NODE_RPC: &str = "http://localhost:20443";
```

Comment:
Since this constant will be reused through the codebase, maybe a better place for its definition might be within the root file definition of the `chainhook-types-rs` library (`./lib.rs`). What you think?

Reply:
I like it, thanks for the suggestion!